recipes-tests/trenchboot-tests/trenchboot-tests/cukinia.conf

```diff
@@ -0,0 +1,74 @@
+export CUKINIA_ROOT=/etc/cukinia
+
+shopt -s nullglob
+
+check_acm() {
+    files=("/boot/acm/$1"*.bin)
+    if [ ${#files[@]} -eq 0 ]; then
+        return 1
+    fi
+    for acm in "${files[@]}"; do
+        if [ -f "$acm" ]; then
+            grep "/acm/$(basename "$1")" /boot/grub/grub.cfg || return 1
+        else
+            return 1
+        fi
+    done
+}
+
+acms=(
+    ADL_SINIT
+    BDW_SINIT
+    CFL_SINIT
+    CMLSTGP_SINIT
+    CML_RKL_S_SINIT
+    CML_S_SINIT
+    RKLS_SINIT
+    SKL_KBL_AML_SINIT
+    SNB_IVB_SINIT
+    TGL_SINIT
+)
+
+cukinia_log "$(_colorize yellow "--- GRUB ---")"
+cukinia_test -f /boot/grub/grub.cfg
+cukinia_test -f /boot/EFI/BOOT/grub.cfg
+as "Check if /boot/grub/grub.cfg and /boot/EFI/BOOT/grub.cfg are the same" \
+    cukinia_cmd cmp /boot/grub/grub.cfg /boot/EFI/BOOT/grub.cfg
+as "grub.cfg contains slaunch" cukinia_cmd grep 'slaunch\s*$' /boot/grub/grub.cfg
+
+cukinia_log "$(_colorize yellow "--- ACM & SKL ---")"
+for acm in "${acms[@]}"; do
+    as "/boot/acm/$acm*.bin exists and is used in grub.cfg" cukinia_cmd check_acm "$acm"
+done
+cukinia_test -f /boot/skl.bin
+as "skl.bin is used in grub.cfg" cukinia_cmd grep "/skl.bin" /boot/grub/grub.cfg
+
+cukinia_log "$(_colorize yellow "--- packages ---")"
+as "Check whether the tpm2_tools package can be started" cukinia_cmd tpm2_eventlog -v
+as "Check whether the rsync package can be started" cukinia_cmd rsync --version
+
+cukinia_log "$(_colorize yellow "--- TPM ---")"
+# cukinia_test -f /sys/kernel/security/tpm0/binary_bios_measurements
+# as "TPM event log can be read by tpm2_eventlog" \
+#     cukinia_cmd tpm2_eventlog /sys/kernel/security/tpm0/binary_bios_measurements
```

Comment:
As part of this project, something similar will be added to the kernel for DRTM measurements, except format may be different (especially if we take older TXT into account) and `tpm2_eventlog` may not know it. However, both existing and new log are just memory regions that aren't handled by TPM, so I'd rather move this to `--- KERNEL ---`.